study/cyber-tutor/quiz/js/app.js

```diff
@@ -32,7 +93,21 @@ function createQuiz() {
 
       // variable to store the list of possible answers
       const answers = [];
-
+	  var arr = [];
+	  var k = 0;
+	  for (letter in currentQuestion.answers) {
+	  arr[k] = currentQuestion.answers[letter];
+	  k = k+1;
+	  }
+	  
+	  arr = randomize(arr);
```

Comment:
You could skip this entire step if you made made the ```answers``` property an array in the ```myQuestions``` object.
```js
{
    question: "What is the relationship between a vulnerability and an exploit?",
    answers: [
      "An exploit is needed for a vulnerability to succeed",
      "A vulnerability enables hackers to utilize an exploit",
      "Hackers can use an exploit even without a vulnerability in a system"
    ],
    correctAnswer: "A vulnerability enables hackers to utilize an exploit"
},
```